vita3k/gui/src/app_context_menu.cpp

```diff
@@ -104,36 +104,34 @@ static bool get_update_history(GuiState &gui, HostState &host) {
     return !update_history_infos[host.io.title_id].first.empty() && !update_history_infos[host.io.title_id].second.empty();
 }
 
-static std::map<std::string, std::string> game_info;
+static std::map<std::string, std::string> app_info;
 
-static bool get_game_info(GuiState &gui, HostState &host) {
-    game_info.clear();
-    const auto game_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
+static bool get_app_info(GuiState &gui, HostState &host) {
+    app_info.clear();
+    const auto app_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
```

Comment:
What is the name of this weird syntax / convention used in `C/C++`?

Reply:
no have understund your ask ?

Reply:
I'm not familiar with `C/C++` so what does this syntax means `{ var0 / var1 / var2 }`? Path concatenation? `{}` means array right? `/` is divide operator right?

Reply:
i have just used code inside io and other part, is same